addons/account/tests/common.py

```diff
@@ -84,13 +85,16 @@ def setUpClass(cls, chart_template_ref=None):
         cls.product_b = cls.env['product.product'].create({
             'name': 'product_b',
             'uom_id': cls.env.ref('uom.product_uom_dozen').id,
+            'uom_po_id': cls.env.ref('uom.product_uom_dozen').id,
             'lst_price': 200.0,
             'standard_price': 160.0,
             'property_account_income_id': cls.copy_account(cls.company_data['default_account_revenue']).id,
             'property_account_expense_id': cls.copy_account(cls.company_data['default_account_expense']).id,
             'taxes_id': [(6, 0, (cls.tax_sale_a + cls.tax_sale_b).ids)],
             'supplier_taxes_id': [(6, 0, (cls.tax_purchase_a + cls.tax_purchase_b).ids)],
         })
+        # vendor bills used to rely on sale uom when past tests were written
+        # so purchase uom is set to sale uom to avoid making the tests fail
```

Comment:
it doesn't make sense to put a comment here :grimacing:
let's remove it, anyway the blame is straightforward :)

Reply:
done